text-ui-test/ACTUAL.TXT

```diff
@@ -0,0 +1,50 @@
+
+  ,--,       ,---,   .--.--.       ,---,
+,--.'|    ,`--.' |  /  /    '.    '  .' \
+|  | :    |   :  : |  :  /`. /   /  ;    '.
+:  : '    :   |  ' ;  |  |--`   :  :       \
+|  ' |    |   :  | |  :  ;_     :  |   /\   \
+'  | |    '   '  ;  \  \    `.  |  :  ' ;.   :
+|  | :    |   |  |   `----.   \ |  |  ;/  \   \
+'  : |__  '   :  ;   __ \  \  | '  :  | \  \ ,'
+|  | '.'| |   |  '  /  /`--'  / |  |  '  '--'
+;  :    ; '   :  | '--'.     /  |  :  :
+|  ,   /  ;   |.'    `--'---'   |  | ,'
+---`-'   '---'                 `--''
+
+Your Lifestyle Scheduling Assistant
+
+type "help" for list of commands
+____________________________________________________________
+
+Everyday is a sunny day!
+____________________________________________________________
+The following task has been added:
+[T][?] add pie crust
```

Comment:
It seems like there's a problem with your TXT files as the ticks and crosses are all shown as `?` - maybe you'd like to investigate further on this issue?

Reply:
Strange, it works on intelliJ,  I think windows cmd does not display unicode properly - have updated the code to work with terminal. Good find!